hpx/util/jenkins_hash.hpp

```diff
@@ -98,8 +99,10 @@ namespace hpx { namespace util
         /// constructors and destructor
         jenkins_hash() : seed_(0) {}
 
+        std::mt19937 gen{(unsigned int)std::random_device{}()};
```

Comment:
Please don't use direct initializations of member variables for now. We have not decided to allow this in HPX yet. Also, I don't think you want to invoke `std:;random_device` for each instance of the `jenkins_hash` type. Creating and using `std::random_device` is a potentially very expensive operation.